react/features/app-settings/components/AppSettings.native.js

```diff
@@ -23,6 +28,17 @@ import { translate } from '../../base/i18n';
  */
 class AppSettings extends AbstractAppSettings {
 
+    /**
```

Comment:
style: unneeded extra newline

Reply:
I insist 😆  I my world class definitions need an extra line after

Reply:
Please look around and be consistent.